server/seeds/index.js

```diff
@@ -1,11 +1,23 @@
 const db = require("../config/connection");
-const { Song } = require("../models");
+const { Song, User, Rating } = require("../models");
 const songSeeds = require("./songSeeds.json");
+const userSeeds = require("./userSeeds.json");
+const ratingSeeds = require("./ratingSeeds.json");
+const songs = Song.create(songSeeds);
 
 db.once("open", async () => {
   try {
     await db.dropDatabase();
-    await Song.create(songSeeds);
+    await songs;
```

Comment:
using await on a variable doesn't make sense. You could just `await Song.create()` here.